exercise_4.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+
+def create_matrix() -> np.array:
+    """
+   A function that return a 10x10 matrix, in which the elements on the borders is equal to 1, and inside 0
+    :return:  a 10x10 matrix, in which the elements on the borders is equal to 1, and inside 0
+    """
+    matrix = np.ones((10, 10))
+    matrix[1:9, 1:9] = 0
```

Comment:
This works great, you can also use numpy.pad
https://numpy.org/doc/stable/reference/generated/numpy.pad.h